sklearn/ensemble/_bagging.py

```diff
@@ -43,12 +43,12 @@ def _generate_indices(random_state, bootstrap, n_population, n_samples):
     return indices
 
 
-def _generate_bagging_indices(random_state, bootstrap_features,
+def _generate_bagging_indices(seed, bootstrap_features,
                               bootstrap_samples, n_features, n_samples,
                               max_features, max_samples):
     """Randomly draw feature and sample indices."""
     # Get valid random state
-    random_state = check_random_state(random_state)
+    random_state = np.random.RandomState(seed)
```

Comment:
We can keep `check_random_state` because it will call the same function under the hood.
But `seed` is a better parameter name in this case.

Reply:
As Guillaume Lemaitre pointed out, `random_state = check_random_state(seed)` looks better.

Reply:
Anyway I leave the function `def _generate_bagging_indices(random_state, ...)` untouched.